Foundation/NSSwiftRuntime.swift

```diff
@@ -22,6 +22,10 @@ import CoreFoundation
 
 @_exported import Dispatch
 
+#if os(Windows)
+import WinSDK
```

Comment:
Where did we end up on this? Will this cause people importing Foundation to also get `WinSDK`?

Reply:
Yeah, I think that currently it does currently result in WinSDK getting pulled in, since we do not yet have the private import (I think that @jrose-apple was working on the that?).  If there is a way to avoid that, I'm happy to try that out.  I don't think that going with Shims for this is very good though because as we start using more interfaces across Windows, we will effectively end up re-creating the SDK.

Reply:
It's probably reasonable in the end; on Darwin it's not like you can't help but import standard Darwin headers if you import Foundation.